src/de/jost_net/JVerein/server/DDLTool/Updates/Update0443.java

```diff
@@ -0,0 +1,68 @@
+/**********************************************************************
+ * Copyright (c) by Heiner Jostkleigrewe
+ * This program is free software: you can redistribute it and/or modify it under the terms of the 
+ * GNU General Public License as published by the Free Software Foundation, either version 3 of the 
+ * License, or (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,  but WITHOUT ANY WARRANTY; without 
+ *  even the implied warranty of  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See 
+ *  the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along with this program.  If not, 
+ * see <http://www.gnu.org/licenses/>.
+ * 
+ * heiner@jverein.de
+ * www.jverein.de
+ **********************************************************************/
+package de.jost_net.JVerein.server.DDLTool.Updates;
+
+import java.sql.Connection;
+
+import de.jost_net.JVerein.server.DDLTool.AbstractDDLUpdate;
+import de.jost_net.JVerein.server.DDLTool.Column;
+import de.jost_net.JVerein.server.DDLTool.Index;
+import de.jost_net.JVerein.server.DDLTool.Table;
+import de.willuhn.util.ApplicationException;
+import de.willuhn.util.ProgressMonitor;
+
+public class Update0443 extends AbstractDDLUpdate
+{
+  public Update0443(String driver, ProgressMonitor monitor, Connection conn)
+  {
+    super(driver, monitor, conn);
+  }
+
+  @Override
+  public void run() throws ApplicationException
+  {
+    Table t = new Table("altersstaffel");
+    Column pk = new Column("id", COLTYPE.BIGINT, 10, null, true, true);
+    t.add(pk);
+    Column beitragsgruppe = new Column("beitragsgruppe", COLTYPE.BIGINT, 10,
+        null, true, false);
+    t.add(beitragsgruppe);
+    Column nummer = new Column("nummer", COLTYPE.BIGINT, 10, null, true,
+        false);
+    t.add(nummer);
+    Column betrag = new Column("betrag", COLTYPE.DOUBLE, 10, null, true,
+        false);
+    t.add(betrag);
+    t.setPrimaryKey(pk);
+    execute(this.createTable(t));
+    
+    Column col = new Column("altersstaffel",
+            COLTYPE.BOOLEAN, 0, "FALSE", true, false);
+    execute(addColumn("beitragsgruppe", col));
+    
+    Index idx = new Index("ixAltersstaffel1", false);
+    idx.add(col);
+    execute(idx.getCreateIndex("beitragsgruppe"));
+    
```

Comment:
Die zwei Zeilen 58/59 sind doch noch falsch. Der Index muss wie der ForeignKey in der Tabelle altersstaffel erzeugt werden. Also so:
idx.add(beitragsgruppe);
execute(idx.getCreateIndex("altersstaffel"));

Reply:
du hast recht, habe ich geändert